src/ccs/lcccs.rs

```diff
@@ -77,13 +86,18 @@ impl<G: Group> LCCCS<G> {
 
   /// Compute all L_j(x) polynomials
   // Can we recieve the MLE of z directy?
```

Comment:
Is this comment still relevant?

Reply:
Good point. I don't think this is needed though. I think we can definitely compute the MLE on the fly as is the only place where this is done.

Reply:
Fixed in 52f6a62